src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -13,6 +13,15 @@ export class ArtworkSidebarPartnerInfo extends React.Component<
   ArtworkSidebarPartnerInfoProps
 > {
   renderPartnerName() {
+    const sale = this.props.artwork.sale
+    if (sale) {
```

Comment:
Wouldn't this end up always displaying the sale name even when the sale is not a benefit auction (e.g. Phillips auctions)?

Reply:
No, because `ArtworkSidebarAuctionPartnerInfo` is used for auctions. This case is only for "buy now" sales which are not marked as auctions.